storage/innobase/log/log0log.cc

```diff
@@ -1098,6 +1098,18 @@ log_group_file_header_flush(
 	srv_stats.os_log_pending_writes.dec();
 }
 
+const char* log_encrypt_name(redo_log_encrypt_enum val) {
+	switch(val) {
+	case REDO_LOG_ENCRYPT_OFF:
+		return "off";
+	case REDO_LOG_ENCRYPT_MK:
+		return "master_key";
+	case REDO_LOG_ENCRYPT_RK:
+		return "rotated_key";
```

Comment:
Although we have chosen the variable name as RK, current terminology is "keyring_key" I think.

Optional comment.

Reply:
Right, I should have changed that `RK` constant, I always mix up things because of it. Fixed.